src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task{
+    protected String eventDetailsPartOne;
+    protected String eventDetailsPartTwo;
+
+    public Event(String description, String eventDetailsPartOne, String eventDetailsPartTwo) {
```

Comment:
eventDetailsPartOne and Two are quite ambiguous - variables named from and to (or variants) would be sufficient.